compiler/quilt/test/test_command.py

```diff
@@ -18,6 +18,106 @@
 from .utils import QuiltTestCase, patch
 
 class CommandTest(QuiltTestCase):
+    # no counterpart to command.get_registry_url, so we mock _save_config and _load_config
+    @patch('quilt.tools.command._save_config')
+    @patch('quilt.tools.command._load_config')
+    @patch('quilt.tools.command.input')
+    def test_config_urls_default(self, mock_input, mock_load_config, mock_save_config):
+        config_prompt = ("Please enter the URL for your custom Quilt registry (ask your administrator),\n" +
+                         "or leave this line blank to use the default registry: ")
+        # test setting default URL with blank string -- result should be default
+        mock_load_config.return_value = {}
+        mock_input.return_value = ''
+        command.config()
+
+        mock_input.assert_called_with(config_prompt)
+
+        args, kwargs = mock_save_config.call_args
+        mock_load_config.return_value = args[0] if args else kwargs['cmd']
+        assert command.get_registry_url() == command.DEFAULT_REGISTRY_URL
+
+    # no counterpart to command.get_registry_url, so we mock _save_config and _load_config
+    @patch('quilt.tools.command._save_config')
+    @patch('quilt.tools.command._load_config')
+    @patch('quilt.tools.command.input')
+    def test_config_good_urls(self, mock_input, mock_load_config, mock_save_config):
+        config_prompt = ("Please enter the URL for your custom Quilt registry (ask your administrator),\n" +
+                         "or leave this line blank to use the default registry: ")
+        test_urls = [
+            'https://foo.com',
+            'http://foo.com',
+            'https://foo.bar.net',
+#REVIEW: is this style of URL intentionally invalid?
```

Comment:
Yes. It's the host for the API calls, so all paths are hard-coded since they're part of the API. (But, it can be http or https, so I had to use an actual URL, not just a host:port string.)

Reply:
Works for me.  I was thinking of the instance where someone has a reverse proxy to their server or some such, and just wanted to be sure.